exporter/fileexporter/testdata/config.yaml

```diff
@@ -0,0 +1,25 @@
+receivers:
+  examplereceiver:
+
+processors:
+  exampleprocessor:
+
+exporters:
+  file:
+  file/2:
+    # This will write the pipeline data to a JSON file.
+    # The data is written in Protobuf JSON encoding
+    # (https://developers.google.com/protocol-buffers/docs/proto3#json).
+    # Note that there are no compatibility guarantees for this format, since it
+    # just a dump of internal structures which can be changed over time.
```

Comment:
nit: add that it is intended to be used when debugging.

Reply:
Fixed.